core/src/components/select/select.tsx

```diff
@@ -562,8 +562,21 @@ export class Select implements ComponentInterface {
   }
 
   private async openAlert() {
-    const label = this.getLabel();
-    const labelText = label ? label.textContent : null;
+    /**
+     * TODO FW-3194
+     * Remove legacyFormController logic.
+     * Remove label and labelText vars
+     * Pass `this.label` instead of `labelText`
+     * when setting the header.
+     */
+    let label, labelText;
```

Comment:
Can we refactor this into two definitions? Having multiple definitions like this defaults the type signature to `any`, instead of:
```ts
let label: string;
let labelText: string;
```